README.md

```diff
@@ -21,9 +22,8 @@ distributed directly to the app namespace.
   * [App Dev/Ops: Consume the infrastructure you need using kubectl](#app-devops-consume-the-infrastructure-you-need-using-kubectl)
   * [APIs in this Configuration](#apis-in-this-configuration)
 * [Customize for your Organization](#customize-for-your-organization)
-* [What's Next](#whats-next)
 * [Learn More](#learn-more)
-* [Local Dev Guide](#local-dev-guide)
+* [Local Dev Guide](development.md)
```

Comment:
i would actually consider removing this development.md file entirely - i think it was useful when we were first bootstrapping these reference platforms, but this initital scrappy file has outlived its usefulness now :)

Reply:
removed